src/model/DataLogger.cpp

```diff
@@ -33,20 +33,23 @@ DataLogger* DataLogger::instance() {
  */
 void DataLogger::loop() {
   unsigned long msNow = millis();
-  if (msNow >= nextSDLogTime) {
-    writeToSD();
-    nextSDLogTime = (msNow / (unsigned long)SD_LOGGING_INTERVAL + 1) * (unsigned long)SD_LOGGING_INTERVAL;
+  if (msNow >= nextDataLogTime) {
+    writeToDataLog();
+    nextDataLogTime = (msNow / DATA_LOGGING_INTERVAL + 1) * DATA_LOGGING_INTERVAL;
```

Comment:
Oh, you're removing the typecasting here and putting it in the constant declaration in the header file. That's good.